lua/gatekeeper/staticlib.lua

```diff
@@ -74,6 +82,48 @@ function alloc_an_lcore (numa_table)
 	return lcore_t[1]
 end
 
+function merge_arrays (a, b)
```

Comment:
`merge_arrays()` is only used in `convert_table_to_array()`, so place `merge_arrays()` just before `convert_table_to_array()`, and add the qualifier `local` to `merge_arrays()`.